internal/validator/handler.go

```diff
@@ -175,7 +175,7 @@ func (this *handler) validateContent(ctx context.Context, requestType string, da
 		if requestType == playbookSatPayloadHeaderValue {
 			err = validateWithSchema(ctx, this.schemas[1], true, line, events)
 			if err == nil {
-			    err = validateSatHostUUID(line)
+				err = validateSatHostUUID(line)
 			}
 		} else {
```

Comment:
I usually run `gofmt -l -w <file_name>`  to avoid weird linting situations. This one might need that. :slightly_smiling_face: